src/lib/components/Logo.svelte

```diff
@@ -0,0 +1,112 @@
+				fill={hex}
+			/>
+			<path
+				d="M74.1513 70.5058L77.1739 73.5576L75.1671 75.4942C73.9134 76.7046 73.8346 78.2723 74.9934 79.4409C76.161 80.6196 77.7694 80.5792 79.0538 79.3386L81.7334 76.7522L74.865 69.817L74.1527 70.5043L74.1513 70.5058ZM80.4009 76.817L78.5546 78.5994C77.5913 79.5288 76.5682 79.5663 75.7159 78.7046C74.881 77.8631 74.9511 76.9222 75.9348 75.9726L77.7913 74.1801L80.4009 76.8156V76.817Z"
+				fill={hex}
+			/>
+			<path
+				d="M58.3608 80.2464C56.0256 80.9942 55.1003 83.4294 56.0431 86.3026C56.986 89.1744 59.1825 90.6124 61.5177 89.8645C63.8398 89.121 64.7783 86.6816 63.8355 83.8084C62.8926 80.9366 60.6829 79.5029 58.3608 80.2464ZM61.2331 88.9942C59.4846 89.5533 57.8223 88.353 57.0443 85.9813C56.265 83.6095 56.8984 81.6744 58.6469 81.1153C60.3822 80.5605 62.0578 81.7565 62.8371 84.1282C63.6151 86.5 62.9685 88.4395 61.2346 88.9942H61.2331Z"
+				fill={hex}
+			/>
+			<path
+				d="M40.1272 82.6081L45.2661 83.0836L44.8895 87.0533L41.2875 86.7205L41.2057 87.5749L44.8093 87.9092L44.5159 90.9971L39.5156 90.5346L39.4339 91.4035L45.4252 91.9582L46.3403 82.3055L40.2089 81.7392L40.1272 82.6081Z"
+				fill={hex}
+			/>
+			<path
+				d="M35.7209 80.2176C33.9315 79.6513 32.2268 80.3617 31.1074 82.1743L31.9597 82.6772C32.8968 81.2435 34.1417 80.6772 35.4378 81.0865C37.1877 81.6412 37.8401 83.5778 37.0695 85.9525C36.2989 88.3271 34.6263 89.5288 32.8895 88.9784C31.6605 88.5893 30.9556 87.4942 30.9731 85.987L29.9617 85.9726C29.9179 87.9222 30.8826 89.304 32.5523 89.8329C34.943 90.5893 37.1366 89.1455 38.0707 86.2709C39.0048 83.3948 38.0576 80.9582 35.7194 80.2176H35.7209Z"
+				fill={hex}
+			/>
+			<path
+				d="M28.146 77.0101L23.6318 84.6081L21.066 83.1225L20.6121 83.8876L26.6136 87.3646L27.0689 86.5994L24.49 85.1066L29.0057 77.5086L28.146 77.0101Z"
+				fill={hex}
+			/>
+			<path
+				d="M51.1538 82.0187C49.2929 82.1383 48.3734 83.1816 48.496 85.0403L48.9806 92L49.9584 91.938L49.5045 85.0303C49.4315 83.7075 49.9686 83.0274 51.1494 82.9524C51.253 82.9452 51.361 82.9438 51.4763 82.9467C52.5899 82.9726 52.8658 83.8429 53 84.6138L54.0086 84.5245C53.9984 84.3314 53.9502 83.8256 53.7064 83.3098C53.4554 82.7824 52.7913 81.915 51.1538 82.0202V82.0187Z"
+				fill={hex}
+			/>
+			<path
+				d="M68.0374 75.6988L70.304 79.3257L68.9145 80.1715L64.2791 77.987L63.2327 78.6239L67.9046 80.7853L67.9615 80.7507C66.4859 81.6484 66.1459 83.0735 67.0259 84.4813C67.8549 85.8084 69.2955 86.2752 70.8893 85.304L74.017 83.3991L68.8824 75.1816L68.0388 75.696L68.0374 75.6988ZM72.7107 83.1772L70.4397 84.5605C69.4049 85.1902 68.4096 84.8084 67.8812 83.9625C67.3529 83.1182 67.3193 82.1484 68.4752 81.4438L70.7579 80.0533L72.7093 83.1772H72.7107Z"
+				fill={hex}
+			/>
+		</svg>
+	</a>
+</div>
+
+<style>
+	div {
+		display: block;
+		position: absolute;
+		left: 30;
+		top: 30;
+		padding: 20px;
```

Comment:
I think we may need a z-index here so it's clickable above the clouds on the main page